olive/cli/convert_adapters.py

```diff
@@ -6,6 +6,9 @@
 from argparse import ArgumentParser
 from typing import TYPE_CHECKING
 
+from onnxruntime import __version__ as ort_version
```

Comment:
onnxruntime is not a dependency of Olive, can you move this import inside the func?